wordpress-mobile/WordPress-Android
# [Gutenberg] share media to Gutenberg

Fixes #9244 

Gutenberg mobile PR ~https://github.com/wordpress-mobile/gutenberg-mobile/pull/758~ (was merged mistakingly by me after getting approval for the Android part)
New PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/763

Most of the work is done on the Gutenberg bridge side, with this particular PR only removing the fallback to Aztec in 10eeefa and updating the Gutenberg mobile hash.

To test:

**Note:** First make sure to have the `Use Block Editor` setting to `ON` in App Settings. Then:

#### CASE A: share from another app

1. open the Photos app
2. pick a picture and tap the share icon
3. select the WordPress app
4. choose a site and tap on `Add to new post`
5. Observe the Gutenberg editor opens up, with a pre-filled `Image Block` and the selected image starts uploading.

![share_media_from_outside](https://user-images.githubusercontent.com/6597771/54575567-4e2cd980-49d3-11e9-8ea6-e95c7ecdd62c.gif)




####  CASE B: start draft from Media Section
1. open the app
2. go to the Media section
3. tap on the `+` icon on the right-top corner of the screen
4. select one option to upload an image to the Media section
5. wait for it to finish uploading
6. once it's uploaded, observe a snackbar appears, with action button `WRITE POST`. Tap on it.
7. observe the Gutenberg editor is fired, with a pre-filled `Image Block` and the selected image starts uploading.

![share_media_from_media_section_snackbar2](https://user-images.githubusercontent.com/6597771/54575813-5c2f2a00-49d4-11e9-8883-d035454c0121.gif)

## EDIT: more test cases were added
Follow description of test cases C, D, E in PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/758

Update release notes:
